azure-pipelines.yml

```diff
@@ -0,0 +1,121 @@
+trigger:
+- v2
+
+jobs:
+
+  - job: V1Test
+    pool: 
+      vmImage: 'windows-latest'
+
+    variables:
+      solution: 'WebJobs.Extensions.DurableTask.sln'
+      buildPlatform: 'Any CPU'
+      buildConfiguration: 'Release'
```

Comment:
Since we're doing a separate build for running tests and doing the release, I wonder if we should actually use `Debug` as the `buildConfiguration` value for the V1 and V2 tests. That might provide us with better diagnostic information (exception traces, etc.) if something fails. We should continue to use the `Release` build configuration for the publish job.